release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+- Following the current {product-title-short} release cadence, each {product-title-short} release will be supported for 9 weeks.
+- The latest stable version of {product-title-short} will support all currently-supported {ocp} releases.
+
+[id="deprecation-notices-40"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed, see the following table. Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+- NA: Not applicable
+
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+| Feature | {product-title-short} 3.73 |{product-title-short} 3.74 | {product-title-short} 4.0
+
+| `AllComments` permission
+| DEP
+| DEP
+| REM
+
+
+| `BuildDate` attribute
+| GA
+| DEP
+| REM
+
+| Clair scanner version 2
+| GA
+| DEP
+| REM
+
+| `ComplianceRuns` permission
+| DEP
+| DEP
+| REM
+
+| `DebugLogs` permission
+| DEP
+| DEP
+| REM
+
+| Examining images for Application-level dependencies for vulnerability reporting: `dotnet/shared/Microsoft.AspNetCore.App/` and `dotnet/shared/Microsoft.NETCore.App/`
+| GA
+| GA
+| DEP
+
+| `Expiration` field in `Exclusion` proto
+| GA
+| GA
+| DEP
+
+| `ids` field in the /v1/cves/suppress` and `/v1/cves/unsuppress` API payload
+a|
+* DEP in {product-title-short} 3.73
+* REM in {product-title-managed-short} (Field Trial)
+| DEP
+| REM
+
+| Kernel module collection method
+| GA
+| DEP
+| DEP
+
+| Network Graph version 1.0
+| GA
+| GA
+| DEP
+
+| `NetworkGraphConfig` permission
+| DEP
+| DEP
+| REM
+
+| `Policy` permission
+| GA
+| DEP
+| REM
+
+| `ProbeUpload` permission
+| DEP
+| DEP
+| REM
+
+| `Role` permission
+| GA
+| DEP
+| REM
+
+| `roxctl scanner generate` flag `offline-mode` (flag only)
+| GA
+| GA
+| DEP
+
+| `ScannerBundle` permission
+| DEP
+| DEP
+| REM
+
+| `ScannerDefinitions` permission
+| DEP
+| DEP
+| REM
+
+| `ScopeManager` default role
+| GA
+| DEP
+| REM
+
+| `SensorUpgradeConfig` permission
+| DEP
+| DEP
+| REM
+
+| `ServiceIdentity` permission
+| DEP
+| DEP
+| REM
+
+| Support for {ocp} versions earlier than 4.10
+| GA
+| DEP
+| REM
+
+| `/v1/cves/suppress` and `/v1/cves/unsuppress`
+a|
+* DEP in {product-title-short} 3.73
+* REM in {product-title-managed-short} (Field Trial)
+| DEP
+| REM
+
+| `/v1/report` endpoint
+| GA
+| GA
+| DEP
+
+| `/v1/serviceaccounts` endpoint
+| GA
+| GA
+| DEP
+
+| `VulnerabilityReports` permission
+| GA
+| DEP
+| REM
+
+| `vulns` field of `storage.Node` object in response payload of `v`/nodes`
```

Comment:
Not sure where the intended backticks are supposed to go, but this one is rendering a backtick verbatim currently:

> `vulns` field of `storage.Node` object in response payload of `v`/nodes`

Reply:
Thank you for catching these - more errors in reformatting after pasting from google doc :(